$MirOS: ports/devel/libglade/patches/patch-glade_glade-xml_c,v 1.2 2005/12/28 18:20:52 bsiegert Exp $
$OpenBSD: patch-glade_glade-xml_c,v 1.1 2004/01/03 05:51:32 marcm Exp $
--- glade/glade-xml.c.orig	Mon Aug 18 07:58:36 2008
+++ glade/glade-xml.c	Thu Oct 18 18:12:47 2012
@@ -1004,7 +1004,7 @@ default_custom_handler(GladeXML *xml, gc
     allsymbols = g_module_open(NULL, 0);
     if (g_module_symbol(allsymbols, func_name, (gpointer)&func))
 	return (* func)(name, string1, string2, int1, int2);
-    g_warning("could not find widget creation function");
+    g_warning("could not find widget creation function: %s", func_name);
     return NULL;
 }
 
